integreat_cms/cms/views/media/media_actions.py

```diff
@@ -129,6 +129,60 @@ def get_query_search_results_ajax(request, region_slug=None):
     return JsonResponse({"data": result})
 
 
+@json_response
+@permission_required("cms.view_mediafile")
+# pylint: disable=unused-argument
+def get_file_usages_ajax(request, region_slug=None):
+    """
+    View to search unused media files
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :return: JSON response with the search result
+    :rtype: ~django.http.JsonResponse
+    """
+
+    file = get_object_or_404(
+        MediaFile.objects.filter(
+            Q(region=request.region) | Q(region__isnull=True, is_hidden=False),
+        ),
+        id=request.GET.get("file"),
+    )
+
+    return JsonResponse({"data": file.serialize_usages()})
```

Comment:
I did however not find a solution to prefetch all usages in a reasonable amount of time, so I went with another dynamic ajax view which only queries the usages when they are expanded in the sidebar.